### PR TITLE
Add support for elasticsearch serverlerless plugin, add initial filte…

### PR DESCRIPTION
**Pull request by ssoper-usgs**
…r patterns to catch exceptions, ERROR, and WARN

Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -29,11 +35,20 @@ custom:
     connectInfo: ${ssm:/aws/reference/secretsmanager/NWCAPTURE-DB-${self:provider.stage}~true}
   observationDb:
     connectInfo: ${ssm:/aws/reference/secretsmanager/WQP-EXTERNAL-${self:provider.stage}~true}
+  esLogs:
+    endpoint: ${ssm:/iow/aws/vpc/${self:provider.stage}/elasticsearchEndpoint}
+    index: aqts-capture-dvstat-transform
```
Comment (ayan-usgs) [inline serverless.yml]: This looks like a copy/paste error.

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "dependencies": {
     "serverless": "2.7.0",
     "serverless-pseudo-parameters": "2.5.0",
```
Comment (ayan-usgs) [inline package.json]: You can remove pseudo-parameters.

**Review by ayan-usgs — CHANGES_REQUESTED** [review]
Nothing major. Just a copy/paste oversight.

File: serverless.yml (hunk 1 above)
Comment (orosz-usgs) [reply serverless.yml]: good catch, I missed that.

Comment (ssoper-usgs) [reply serverless.yml]: Thank you, updated to use service name

File: package.json (hunk 2 above)
Comment (ssoper-usgs) [reply package.json]: Will do.

**Pull request merged**